### ISSUE TITLE
🟡 [High] 時系列交差検証が未実装（過学習検出が不十分）

### ISSUE THREAD

**Issue by kenta10403** [kenta10403/ai-fishing-forecast#36]
## 問題内容

`train_real_marine.py:139` で `shuffle=False` を設定して時系列を崩さないようにしているが、**時系列交差検証（TimeSeriesSplit）を使っていない**ため、未来予測の汎化性能が不明。

## 影響

- 過学習の検出が甘い（たまたまテストデータの期間が予測しやすかった可能性）
- 異なるデータ期間での再現性検証ができない
- モデルの真の予測能力が測定できていない

## 証拠

- `train_real_marine.py:139`: `train_test_split(X, y, test_size=0.2, shuffle=False)`
- コメントに「とりあえずベースライン」と記載あり

### 1. TimeSeriesSplit を導入

```python
from sklearn.model_selection import TimeSeriesSplit

tscv = TimeSeriesSplit(n_splits=5)
scores = []
for train_index, test_index in tscv.split(X):
    X_train, X_test = X.iloc[train_index], X.iloc[test_index]
    y_train, y_test = y.iloc[train_index], y.iloc[test_index]
    model.fit(X_train, y_train)
    scores.append(r2_score(y_test, model.predict(X_test)))
print(f"CV平均R²: {np.mean(scores):.4f} ± {np.std(scores):.4f}")
```

### 2. 複数の評価期間でテスト
- 2023年データで学習 → 2024年で評価
- 2022-2023年で学習 → 2024年で評価
- 季節ごとの精度差を確認

### 3. 残差分析
予測誤差が大きい日付を抽出し、共通パターン（台風、黒潮蛇行など）を調査

## 優先度
🟡 **High** - モデル評価の信頼性に関わる

**Comment by kenta10403** [comment]
PR #52 のマージにより TimeSeriesSplit が導入されました。解決済みです。
